controls/sql.sp

```diff
@@ -17,20 +17,23 @@ variable "sql_db_instance_avg_cpu_utilization_high" {
 }
 
 locals {
-  sql_common_tags = merge(local.thrifty_common_tags, {
-    service = "sql"
+  sql_common_tags = merge(local.gcp_thrifty_common_tags, {
+    service = "GCP/SQL"
```

Comment:
```suggestion
    service = "GCP/CloudSQL"
```
?

Reply:
@rajlearner17 Since all GCP services in the API have the word "Cloud" to help differentiate them from other Google APIs, like Books, YouTube, etc., we can remove `Cloud` from the service names to remove the redundancy; however, if `Cloud` is required to disambiguate a service or is central to its meaning, then we can keep it in.